packages/standalone/capabilities/bundle/config/values.yaml

```diff
@@ -1,6 +1,7 @@
 #@data/values
 ---
 
+namespace:
```

Comment:
question: do we want to provide a default value for ns ? Wondering if ytt will assign `namespace: null` to the Deployment.metadata.namespace if nothing is provided.

Reply:
i don't have any preference, we left the default namespace empty for other packages too.. resolving this comment to keep this package consistent with other packages in the repo